app/core/ingestion/processor.py

```diff
@@ -25,49 +25,48 @@
 ProcessFunc = Callable[[IngestContext, _RowType], None]
 
 
+def _get_ingest_operation(has_new_document: bool) -> IngestOperation:
+    # TODO : Add logic for DELETE
+    if not has_new_document:
+        return IngestOperation.CREATE
+    if has_new_document:
+        return IngestOperation.UPDATE
+    raise TypeError("Cannot determine operation for row")
```

Comment:
This does seem like overkill, will the operations not always be create or update? (delete is just an update after all)

Reply:
Removed